flashCards.rb

```diff
@@ -0,0 +1,123 @@
+require "pry"
+class Flashcard 
+	attr_accessor :front, :back
+	def initialize
+
+	@front = "front"
+    @back = "back"
+    @new_front = []
+    @new_back = []
+
+    @@default_cards = [@front, @back]
+    @@new_cards = [@new_front, @new_back]
```

Comment:
This will give you a class variable that is an array containing all instances of Flashcards, which is what I think you were going for here.